src/rtk/features/overviewOrg/overviewOrgSlice.ts

```diff
@@ -181,10 +229,23 @@ const overviewOrgSlice = createSlice({
       }
     },
   },
+  extraReducers: (builder) => {
+    builder
+      .addCase(fetchOverviewOrgs.fulfilled, (state, action) => {
+        const dataArray = action.payload;
+        const responseList: OverviewOrg[] = mapToOverviewOrgList(dataArray, state.layout);
```

Comment:
collect the layoutState from action instead

Reply:
I just used it directly instead 🙂